stream-chat-android-docs/src/main/java/io/getstream/chat/docs/java/Android.java

```diff
@@ -257,6 +257,12 @@ public void onStopTyping() {
             });
         }
 
+        public void customMaxMessageLengthHandler() {
+            messageInputView.setMaxMessageLengthHandler((messageText, messageLength, maxMessageLength, maxMessageLengthExceeded) -> {
```

Comment:
Would be nice to mention this handler in docusaurus

Reply:
Thought this handler doesn't deserve to be in the docs. But yes, agree, as Niko said we should be on the side of over-explaining 👍

Reply:
I think we shouldn't create a whole new section for this handler but rather add it to a place where we list other ones